test/client.jl

```diff
@@ -1,8 +1,24 @@
-using HTTP, Test, JSON
```

Comment:
these should not have been deleted.
I believe they are here so that this test can be run via
```
julia --project=. test/client
```

Reply:
I wasn't able to find concrete answers to this, and how people test in Julia.

Personally I've just been going into the the REPL and running all tests from there. The `runtests.jl` file also contains all the deps and usages there, which makes things nice and clean.

But as you point out, if you want to run individual test files, they won't work. Maybe containing all deps in their respective test files is the best way of doing this?

Current form:
```julia
test/runtests.jl
using HTTP, Test, JSON
include("test/my_test.jl")
```

Proposed form:
```julia
```

Reply:
The Current form, for *this file* at-least is the proposed form.
Changing that is beyond the scope of this PR.

Reply:
I have a WIP branch currently for re-factoring testing in this package. I'll revert this change (as I think I think I accidentally included it during a rebase), and when that CR goes up we can address this there.